tests/click_tests/recovery.py

```diff
@@ -0,0 +1,56 @@
+from .. import buttons
+
+
+def enter_word(debug, word):
+    word = word[:4]
+    for coords in buttons.type_word(word):
+        debug.click(coords)
+    return debug.click(buttons.CONFIRM_WORD, wait=True)
+
+
+def select_number_of_words(debug, num_of_words=20):
+    # confirm recovery
+    layout = debug.wait_layout()
+    assert layout.text.startswith("Recovery mode")
+    layout = debug.click(buttons.OK, wait=True)
+
+    # select number of words
+    assert "Select number of words" in layout.text
+    layout = debug.click(buttons.OK, wait=True)
+    assert layout.text == "WordSelector"
+
+    if num_of_words != 20:
```

Comment:
```python
word_option_offset = 6
word_options = (12, 18, 20, 24, 33)
index = word_option_offset + word_options.index(num_of_words)  # raises if num of words is invalid
coords = buttons.grid34(index % 3, index // 3)
```
:)

Reply:
I have to admit I was a bit lazy there. Fixed.